.github/workflows/verify_rc.yml

```diff
@@ -234,13 +255,32 @@ jobs:
       VERSION: ${{ needs.target.outputs.version }}
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
       - name: Run
         run: |
           dev/release/verify-release-candidate.sh ${VERSION} ${RC}
       - name: Verify the previous version
+        # TODO: We may re-enable this in the future.
+        # There are some problems for now:
+        # * x86_64: libLLVM.so.18.1 needed by gandiva2000-libs on AlmaLinux 9
+        # * arm64: libarrow.so.2000.0.0: refers the
+        #   "std::condition_variable::wait(std::unique_lock<std::mutex>&)@GLIBCXX_3.4.30"
+        #   (not "...@@GLIBCXX_3.4.30" nor "...@GLIBCXX_3.4.11") symbol on
+        #   AmazonLinux 2023.
```

Comment:
I think that this is not our problem. I think that Amazon Linux 2023 changed `g++` or `g++` related packages. (FYI: `libarrow.so.2100.0.0` refers `...@GLIBCXX_3.4.11` but we didn't change `g++`.)